### ISSUE TITLE
Cannot transition to state 'terminating'

### ISSUE THREAD

**Issue by TristanWright** [Kitware/cumulus#277]
When trying to terminate an EC2 Cluster with an "error" state.

```
"Cluster is in state error and cannot transition to state terminating"
```

**Comment by kotfic** [comment]
@TristanWright just guessing,  but is this related to https://github.com/ansible/ansible-modules-core/pull/4012?

**Comment by TristanWright** [comment]
I don't think so? A little searching here shows that an ['error' status can only transition to 'error.'](I don't think so? ) Is that related?

**Comment by kotfic** [comment]
Yes,  I see what you're saying!  I guess I thought this should be the expected behavior,  if there is an error then it probably requires manual intervention to resolve it.  I think we can just add terminating to the list of valid transitions yeah?

**Comment by cjh1** [comment]
@kotfic Yes, I think that is the correct fix.

**Comment by TristanWright** [comment]
closed with #278
